docker/Dockerfile

```diff
@@ -0,0 +1,64 @@
+ARG BUILDER_IMAGE=python:3.9-slim-bullseye
```

Comment:
This image doesn't include ROOT to make things smaller. If the dev team would prefer to have a base image with ROOT I can do that. Also, I'm assuming the dev team doesn't care if the base image is Debian or CentOS based, but if for some reason it does matter let me know.

Reply:
Hi @matthewfeickert, thanks for all this! Having root would be great, that is generally the most problematic part for the users and our [public analysis database](http://madanalysis.irmp.ucl.ac.be/wiki/PublicAnalysisDatabase) mostly consist of Delphes based analyses for the time being so root is quite essential. The base is not so important ma5 has its own interface and the user doesn't need to know much about shell so we can handle the installation of the tools in the background. Also, you don't need to worry about the update notes I can take care of that, you are right it needs updating.

Reply:
> Having root would be great, that is generally the most problematic part for the users and our [public analysis database](http://madanalysis.irmp.ucl.ac.be/wiki/PublicAnalysisDatabase) mostly consist of Delphes based analyses for the time being so root is quite essential.

Okay cool. :+1: I'll do some revision and then rebase this PR with one of the ATLAS AMG lab images that contain ROOT (I maintain these to try to be decently small images, but you can be the judge on how big is "too big" for an image with `madanalysis5`).